### DISCUSSION TITLE
Endpoints to get the list of user groups

### DISCUSSION THREAD

**Discussion by joomlapi** [joomla/joomla-cms#35080] [Q&A] [answered]
Hello,

Unless I am mistaken, I did not find an endpoint allowing to retrieve the list of user groups (I am talking about groups like Public, Guest, Registered, ...).

It doesn't make sense to me that this endpoint doesn't exist considering that a group ID is needed to use other endpoints.

I accidentally tried this and other derivatives: {{base_path}}/api/index.php/v1/users/groups but no luck.

Is there an endpoint allowing to retrieve the list of user groups? Any other way to do it?

I am basing myself on this documentation:
https://docs.joomla.org/J4.x:Joomla_Core_APIs

Is there any other documentation elsewhere on the subject of web services for Joomla 4?

Finally, I'm sorry to put my questions in this tracker; Is there a better forum for talking about web services? I did not find anything in this regard ...

Sincerely,
Eric.

**Comment by alikon** [comment, marked as answer]
see #30210 should be available on 4.1

**Comment by joomlapi** [comment]
Great, thank you !<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35050">issues.joomla.org/tracker/joomla-cms/35050</a>.</sub>
